config/staging/jenkins.patch

```diff
@@ -217,7 +217,7 @@ index f32806a..51c5c79 100644
        <mode>NORMAL</mode>
        <retentionStrategy class="hudson.slaves.RetentionStrategy$Always"/>
        <launcher class="hudson.slaves.JNLPLauncher"/>
--      <label>windows xp vista 7 8 8.1 32bit 64bit endurance</label>
+-      <label>windows xp 7 8.1 32bit 64bit endurance</label>
 -      <nodeProperties/>
 -      <userId>anonymous</userId>
 +      <label>windows 8.1 64bit endurance</label>
```

Comment:
Do we still run endurance? I suspect this label can go, but that would be a separate patch.